<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>房间</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="targettempup">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/按钮？.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>390</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
border-image: url(:/back.jpg);</string>
    </property>
    <property name="text">
     <string>退房</string>
    </property>
   </widget>
   <widget class="QLabel" name="currenttemp1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/indoor.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="AC_onoff">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/onoff.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="room">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/room.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="instruction">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>40</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
border-image: url(:/back.jpg);
</string>
    </property>
    <property name="text">
     <string>空调使用说明</string>
    </property>
   </widget>
   <widget class="QLabel" name="targettemp1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/set.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>290</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="radio_low">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>低</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_middle">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>中</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_high">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>高</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="targettempdown">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/按钮down.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="currentspeed1">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>290</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/feng.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="CA_state1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>76</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ccair.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="changerequest">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>272</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ok.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="room_num">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CCstate">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="current_temp">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="target_temp">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>230</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CCmodel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="current_cost">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>410</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>370</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/price.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="back">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/1f178a82b9014a900c577a71a8773912b21bee95.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="air">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>381</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/back.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>71</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/model.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>back</zorder>
   <zorder>air</zorder>
   <zorder>targettempup</zorder>
   <zorder>exit</zorder>
   <zorder>currenttemp1</zorder>
   <zorder>AC_onoff</zorder>
   <zorder>room</zorder>
   <zorder>instruction</zorder>
   <zorder>targettemp1</zorder>
   <zorder>targettempdown</zorder>
   <zorder>currentspeed1</zorder>
   <zorder>CA_state1</zorder>
   <zorder>changerequest</zorder>
   <zorder>room_num</zorder>
   <zorder>CCstate</zorder>
   <zorder>current_temp</zorder>
   <zorder>target_temp</zorder>
   <zorder>current_cost</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>CCmodel</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
